### PR TITLE
Check absinthe dependent libraries on CI runs

### PR DESCRIPTION
**Pull request by bryanjos**
This PR adds a GitHub action to check the libraries that depend on Absinthe to ensure they still work with the changes added in PRs.

### CODE REVIEW

**Comment by benwilson512** [comment]
Oh this reminds me, we should update the Elixir versions we test against, particularly with 1.17 out. We should only do 1.15, 1.16, and 1.17

File: .github/workflows/elixir.yml [hunk 1]
```diff
@@ -14,22 +14,21 @@ jobs:
...
+          - '1.17'
         otp:
-          - "24"
-          - "25"
-          - "26"
-        include:
-          - elixir: "1.15"
-            otp: "26"
-            format: true
+          - '24'
```
Comment (benwilson512) [inline .github/workflows/elixir.yml:21]: I'm fine dropping this one at this point.

**Comment by maartenvanvliet** [comment]
Is it failing because of the change/fix in the `String.jaro_distance/2` in Elixir? https://github.com/elixir-lang/elixir/pull/13369

**Comment by bryanjos** [comment]
@maartenvanvliet thanks for that! I've been trying to figure it out. Especially since it passes locally with Elixir 1.17 and OTP 27, though I'm on a mac. I'll look and see if that's the issue.

**Comment by bryanjos** [comment]
@benwilson512 I think this is good to go now! Aside from the CI changes, I lowered the jaro_distance slightly (thanks again @maartenvanvliet!), and fixed an issue dialyzer found.

**Review by benwilson512 — APPROVED** [review]
Really love how clean this is, thanks!

**Comment by maartenvanvliet** [comment]
> @benwilson512 I think this is good to go now! Aside from the CI changes, I lowered the jaro_distance slightly (thanks again @maartenvanvliet!), and fixed an issue dialyzer found.

No problem, saw the CI error and vaguely recalled the change in elixir a few months back

**Pull request merged**